app/resources/game_content.py

```diff
@@ -0,0 +1,203 @@
+    if not elements["gpu"] in hardware["gpu"].keys():
+        return False, {"error": "element_gpu_not_found"}
+    if not elements["motherboard"] in hardware["mainboards"].keys():
+        return False, {"error": "element_motherboard_not_found"}
+    for disk in elements["disk"]:
+        if disk not in hardware["diskStorage"].keys():
+            return False, {"error": "element_disk_not_found"}
+    for ram in elements["ram"]:
+        if ram not in hardware["ram"].keys():
+            return False, {"error": "element_ram_not_found"}
+
+    return True, {}
+
+
+def check_compatible(elements: dict) -> Tuple[bool, dict]:
+    exists, message = check_elements_exist(elements)
+    if not exists:
+        return False, message
+
+    cpu: str = elements["cpu"]
+    motherboard: str = elements["motherboard"]
+    ram: List[str] = elements["ram"]
+    gpu: str = elements["gpu"]
+    disk: List[str] = elements["disk"]
+
+    if hardware["cpu"][cpu]["sockel"] != hardware["mainboards"][motherboard]["sockel"]:
+        return False, {"error": "cpu_and_mainboard_sockel_do_not_fit"}
+
+    if hardware["mainboards"][motherboard]["ram"]["ramSlots"] < len(ram):
+        return False, {"error": "mainboard_has_not_this_many_ram_slots"}
+
+    for ram_stick in ram:
+        if hardware["ram"][ram_stick]["ramTyp"] != hardware["mainboards"][motherboard]["ram"]["typ"]:
+            return False, {"error": "ram_type_does_not_fit_what_you_have_on_your_mainboard"}
+
+    for i in disk:
+        if hardware["diskStorage"][i]["interface"] != hardware["mainboards"][motherboard]["diskStorage"]["interface"]:
+            return False, {"error": "your_hard_drive_interface_does_not_fit_with_the_motherboards_one"}
+
+    return True, {}
+
+
+def calculate_power(elements: dict) -> Tuple[float, float, float, float, float]:
+    cpu: str = elements["cpu"]
+    motherboard: str = elements["motherboard"]
+    ram: List[str] = elements["ram"]
+    gpu: str = elements["gpu"]
+    disk: List[str] = elements["disk"]
+
+    performance_cpu: float = hardware["cpu"][cpu]["cores"] * hardware["cpu"][cpu]["frequencyMax"]
+
+    performance_ram: float = 1
+    for ram_stick in ram:
+        performance_ram += (
+            min(
+                resolve_ram_type[hardware["mainboards"][motherboard]["ram"]["type"]],
+                resolve_ram_type[hardware["ram"][ram_stick]["ramTyp"]],
+            )
+            * hardware["ram"][ram_stick]["ramsize"]
+        )
+
+    performance_gpu: float = resolve_gpu_type[hardware["graphiccards"][gpu]["interface"]] * math.sqrt(
+        hardware["graphiccard"][gpu]["ramsize"] * hardware["graphiccard"][gpu]["frequency"]
+    )
+
+    dick_storage: float = 1
+
+    for i in disk:
+        dick_storage += hardware["diskStorage"][i]["capacity"] * math.log1p(
+            hardware["diskStorage"][i]["writingSpeed"] * hardware["diskStorage"][i]["readingSpeed"]
+        )
+
+    network: float = hardware["mainboards"][motherboard]["speed"]
```

Comment:
dict has no key `"speed"`
```suggestion
    network: float = hardware["mainboards"][motherboard]["networkCard"]["speed"]
```